data-access/cert-api-client.js

```diff
@@ -46,6 +46,66 @@ const postDataDictionaryResultsToApi = async ({
   }
 };
 
+const postWebAPIResultsToApi = async ({
+  url,
+  providerUoi,
+  providerUsi,
+  recipientUoi,
+  webAPIReport = {}
+} = {}) => {
+  if (!url) throw new Error('url is required!');
+  if (!providerUoi) throw new Error('providerUoi is required!');
+  if (!providerUsi) throw new Error('providerUsi is required!');
+  if (!recipientUoi) throw new Error('recipientUoi is required!');
+  if (!Object.keys(webAPIReport)?.length) throw new Error('web API report is empty!');
+
+  try {
+    const { id: reportId = null } =
+      (
+        await axios.post(
+          `${url}/api/v1/certification_reports/web_api_server_core/${providerUoi}`,
+          webAPIReport,
+          {
+            maxContentLength: Infinity,
+            maxBodyLength: Infinity,
+            headers: {
+              Authorization: `ApiKey ${CERTIFICATION_API_KEY}`,
+              recipientUoi,
+              'Content-Type': 'application/json',
+              providerUsi
+            }
+          }
+        )
+      ).data || {};
+
+    if (!reportId) throw new Error('Did not receive the required id parameter from the response!');
+
+    return reportId;
+  } catch (err) {
+    throw new Error(`Could not post web API results to API! ${err}`);
+  }
+};
+
+const deleteDataDictionaryResults = async ({ url, reportId } = {}) => {
```

Comment:
Deleting reports to add new ones was removed in #34 and there is no more `--overwrite` option.

We can leave this code here, but should hook it up to a `--deleteExistingReport` option instead.